app/views/vm_common/_snap.html.haml

```diff
@@ -6,7 +6,7 @@
   #form_div
     %h3= _('Snapshot Information')
     .form-horizontal
-      .form-group
+      .form-group{"hidden" => @record.vendor == "redhat" ? true : nil }
```

Comment:
@blomquisg can we use the SupportsFeature mixin instead of adding yet another assumption in the UI?